tox.ini

```diff
@@ -1,11 +1,12 @@
 [tox]
-envlist = py27, py27-cl, py36
+envlist = py27, py27-cl, py35
```

Comment:
I changed this to Python 3.5 because it's the default in Ubuntu 16.04 LTS onwards. The upcoming 17.10 will change to 3.6 but I think we should try to be compatible with the lowest 3.x version possible.

Reply:
There doesn't seem to be any good way to install 3.5 on macs.
Also, I don't think we should drop 3.6 completely.

Practically, requiring both will make one platform fail on macs, and the other on linux, and both test on travis. Good enough?

Reply:
Yes :)